src/moj/components/ticket-panel/_ticket-panel.scss

```
@@ -0,0 +1,36 @@
+/* ==========================================================================
+   #TICKET PANEL
+   ========================================================================== */
+
+.moj-ticket-panel {
+   display: block;
+   margin-right: govuk-spacing(0);
+   flex-wrap: wrap;
+
+   &--inline {
+      @include govuk-media-query($from: tablet) {
+         display: flex;
+         margin-right: govuk-spacing(3);
+         flex-wrap: nowrap;
+   
+         & > * + * {
+            margin-left: govuk-spacing(3);
+         }
+      }
+   }
+
+   &__content {
+      display: block;
+      position: relative;
+      background-color: govuk-colour("light-grey");
+      padding: govuk-spacing(3);
+      margin-bottom: govuk-spacing(3);
+      border-left: 5px solid $govuk-brand-colour;
+      
+      &--red { border-left: 5px solid govuk-colour("red"); }
```

We only really need to specify the colour here:
```suggestion
      &--red { border-left-color: govuk-colour("red"); }
```

---

Changed these to border-color instead